dispatcher.go

```diff
@@ -335,7 +335,7 @@ func (d *Dispatcher) Start() error {
 			errs = append(errs, newErrors...)
 		}
 
-		return errors.ErrorGroup(errs)
+		return multierr.Combine(errs...)
```

Comment:
If we're exposing multierr to the user outside of x/, than we're creating a contract.

Given multierr is not 1.0, are we sure we want to commit in this way? We might get into a position where we can no longer evolve multierr, or get stuck on 0.1 here.